.github/workflows/lint-test.yaml

```diff
@@ -24,5 +24,13 @@ jobs:
       - name: Set up chart-testing
         uses: helm/chart-testing-action@v2.0.1
 
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --config ct.yaml)
```

Comment:
remove the `--config ct.yaml` since we did not configure it.

Reply:
have fixed, thanks for the review!